cmd/thanos/compact.go

```diff
@@ -737,6 +741,14 @@ func (cc *compactConfig) registerFlag(cmd extkingpin.FlagClause) {
 	cmd.Flag("hash-func", "Specify which hash function to use when calculating the hashes of produced files. If no function has been specified, it does not happen. This permits avoiding downloading some files twice albeit at some performance cost. Possible values are: \"\", \"SHA256\".").
 		Default("").EnumVar(&cc.hashFunc, "SHA256", "")
 
+	cc.filterConf = &store.FilterConfig{}
+
+	cmd.Flag("min-time", "Start of time range limit to serve. Thanos Compactor will serve only metrics, which happened later than this value. Option can be a constant time in RFC3339 format or time duration relative to current time, such as -1d or 2h45m. Valid duration units are ms, s, m, h, d, w, y.").
```

Comment:
Please update the description `Thanos Compactor will serve only metrics`. `serve` is not a suitable verb for compactor.